docs/dev/annual_updates.rst

```diff
@@ -156,57 +142,140 @@ B. FERC Form 1
+    from the XBRL data therefore allowing us to merge the data into one continuous
+    timeseries. The ``row_literal`` column is the DBF label for the ``row_number`` in
+    question. This ``row_literal`` must be mapped to an ``xbrl_factoid`` from the XBRL
+    data. These ``xbrl_factoid`` entires are the value columns from the raw XBRL data.
+
+    Look at the ``row_literal`` values for a given table and see which XBRL columns they
+    coorespond to. It's helpful to
+    `view the XBRL taxonomy <https://xbrlview.ferc.gov/>`__ for the table in question.
+
+    The ``row_literals`` may contain elements of the FERC 1 form such as
+    headers that don't map to an XBRL column. These can be marked as ``headers`` in the
+    ``row_type`` column. Other values are either marked as ``report_value`` or
+    ``calculated_value`` to indicate whether the XBRL data is original or can be
+    calculated from other fields. The way to check this is by looking at the online
+    taxonomy and clicking on the "Relationships" tab for a given field. If there are
+    relationships it's probably a ``calculated_value`` and if there are not, it's a
+    ``reported_value``
+
+    The ``dbf_only`` column is marked ``TRUE`` if the ``row_literal`` only shows up in
+    the DBF files. An common example is when several fields are aggregated in the DBF
+    data but not in XBRL. The ``notes`` column is a place to indicate complexity or
+    reasoning and is intended for humans (vs. computers) to read.
+
+
+**4.B.2)** Use the FERC 1 debugging notebook ``devtools/ferc1-etl-debug.ipynb`` to run
+the transforms for each table. Heed any errors or warnings that pop up in the
+logs. One of the most likely bugs will be uncategorized strings. Any column listed in
+:py:const:`pudl.transform.params.ferc1.TRANSFORM_PARAMS` under ``categorize_strings``
+gets cleaned according to varables defined in :mod:`pudl.transform.params.ferc1`.
+Uncategorized string values (think new, strange fuel type spellings) will throw an error
+and should be added to the appropriate dictionaries in
+:mod:`pudl.transform.params.ferc1`.
+
+**4.B.3)** If there's a new column, add it to the transform process. At the very least,
+you'll need to include it in the ``rename_columns`` dictionary in
+:py:const:`pudl.transform.params.ferc1.TRANSFORM_PARAMS` for the appropriate table.
+
+* Consider whether the column could benefit from any of the standard transforms in
+  :mod:`pudl.transform.classes`. If so, add those transforms/the new column to
+  :py:const:`pudl.transform.params.ferc1.TRANSFORM_PARAMS`. If the standard transform
+  you're applying to the new column was not previously listed in
+  :py:const:`pudl.transform.params.ferc1.TRANSFORM_PARAMS` under the table you're
+  working on, you'll need to add it to ``transform_main()`` in
+  :mod:`pudl.transform.ferc1` under the cooresponding table transformer class or it
+  won't be executed.
```

Comment:
I'm not sure this is true. Most of the things in `Ferc1AbstractTableTransformer.transform_main` are do-nothing-ers if there are no parameters defined in `TRANSFORM_PARAMS`.

When I add a param to a table's `TRANSFORM_PARAMS`, I double check that it gets run in whatever my current iteration of my table-transformer. I usually do this by finding the where the transformer method is being called in the high-level methods of `Ferc1AbstractTableTransformer` (i.e. `process_dbf`, `process_xbrl`, `transform_start`, `transform_main` and `transform_end`). Once I've determined where it is getting called in those high-level methods, I determine if that specific transform stage is being overwritten in the corresponding table transformer and if so is it being overwritten in a way in which the default transform step is or is not being run (is `self.super()` being used?).

Reply:
You're right -- I understand this process much better now and updated it!